src/platal_map.h

```diff
@@ -31,6 +31,7 @@ class Map {
     int width_, height_; // width and height of the map, in squares
     int** map_array_; // loaded from csv
     std::map <Position, Object> *objects_; // map from position on screen to object
+    std::map <Position*, Object> *npc_; // map from pointers to position on screen to object
```

Comment:
Are you sure it's `Position*`?

Reply:
Yeah, I think it should be position* because we want it to point to the posisiton of the npc...
that way the thing tracks npc movement

Reply:
Then `(*npc_)[npc.GetPosition()]= npc;` is kinda wrong